InstaBotPrototype/InstaBotPrototype/Controllers/AuthenticationController.cs

```diff
@@ -0,0 +1,20 @@
+﻿using Microsoft.AspNetCore.Mvc;
+
+namespace InstaBotPrototype.Controllers
+{
+    [Route("api/[controller]")]
+    public class AuthenticationController : Controller
+    {
+        [HttpGet]
+        int Login([FromBody]string login, [FromBody]string password)
+        {
+            return 0;
```

Comment:
If this is controller, why not to return some well known ActionResult or smth like this, more typical for controller methods return values.
Why to return int?